android/src/main/java/com/reactlibrary/RNSpotifyRemoteAuthModule.java

```diff
@@ -45,6 +45,7 @@ public RNSpotifyRemoteAuthModule(ReactApplicationContext reactContext) {
     public void authorize(ReadableMap config, Promise promise) {
         mConfig = config;
         String clientId = mConfig.getString("clientID");
+        String responseType = mConfig.getString("responseType");
```

Comment:
Only a suggestion: AuthorizationResponse.Type responseType =
                AuthorizationResponse.Type.valueOf(mConfig.getString("responseType"));

then on the old line 59 you can just do
AuthorizationRequest.Builder builder = new AuthorizationRequest.Builder(
                clientId,
                responseType,
                redirectUri
        );